src/w3d/renderer/dx8renderer.cpp

```diff
@@ -724,17 +724,41 @@ unsigned int DX8TextureCategoryClass::Add_Mesh(Vertex_Split_Table &split_table,
                     if (Compare_Materials(split_table.Peek_Material(i, pass), m_material)) {
                         if (split_table.Peek_Shader(i, pass) == m_shader) {
                             unsigned short index = vertex_offset + polys[i].I;
-                            std::clamp(index, vmin, vmax);
+
+                            if (vmin >= index) {
```

Comment:
This is updating the min and max you would normally do this with a function to make it clearer:
```
vmin = std::min(index, vmin);
vmax = std::max(index, vmax);
```